docs/en_US/reference/experiment_config.rst

```diff
@@ -538,10 +538,10 @@ If `trialGpuNumber`_ is less than the length of this value, only a subset will b
 This will be used as ``CUDA_VISIBLE_DEVICES`` environment variable.
 
 
-trialPrepareCommand
+pythonPath
 -------------------
 
-Command(s) to run before launching each trial.
+The path will insert at the front of PATH.
```

Comment:
suggest to add detailed description here, for example, what is this field used for? how to set the path for different python environments and OS, add examples like the ones you wrote in the example yaml file below

Reply:
Done.